examples/c/ex_all.c

```diff
@@ -1373,5 +1373,23 @@ main(int argc, char *argv[])
         (void)crc32c;
     }
 
+    {
+        const char *buffer = "some other larger string";
```

Comment:
These should be in size order like WT's normal coding standard. So this should be at the bottom of the variable list.

Reply:
Fixed in https://github.com/wiredtiger/wiredtiger/pull/9900/commits/895b8d0771f065c19e678d9125a9684fcc9bd02c